src/test/java/org/datadog/jenkins/plugins/datadog/clients/DatadogClientTest.java

```diff
@@ -37,6 +39,9 @@ of this software and associated documentation files (the "Software"), to deal
 
 public class DatadogClientTest {
 
+    @ClassRule
+    public static JenkinsRule j = new JenkinsRule();
```

Comment:
Do we need this ?

Reply:
Yes, without it one of the tests fail because a Jenkins instance doesn't get created.